kubetest2-gke/deployer/down.go

```diff
@@ -86,7 +86,7 @@ func (d *Deployer) DeleteClusters(retryCount int) {
 func (d *Deployer) DeleteCluster(project, loc string, cluster cluster) {
 	if err := runWithOutput(exec.Command(
 		"gcloud", containerArgs("clusters", "delete", "-q", cluster.name,
-			"--project="+project,
+			"--project="+project, fmt.Sprintf("--timeout=%d", int(d.DownTimeout.Seconds())),
```

Comment:
why defaulting and not append only if set? that will be completely backwards compatible